### ISSUE TITLE
[Angular UI] Constructors are being called twice on reload page.

### ISSUE THREAD

**Issue by begandroide** [abpframework/abp#10781]
Hi! recently I noted an issue: when I reload the page, then constructor is called twice.

* **ABP Framework version**: 4.4.4
* Angular UI (12)
* EF Core

### Steps to reproduce the problem

- Create default project of abp with angular UI
- Add console.log("home") in constructor of home.component.ts
- Reload the page
- You will see in console that console log is being logged twice.

**note**: this doesn't happens if you navigate from one module to another.
![bug-constructor](https://user-images.githubusercontent.com/22018004/143948025-c63c4dfe-a2f4-40d0-b518-4190acc800de.gif)

**Comment by mehmet-erim** [comment]
Hello @begandroide

We've fixed the initialization problem. However, navigation between layouts (like from empty layout to application layout) still the same issue exists. 
I suggest you move the logic from the constructor to the `ngOnInit`.

Thanks for reporting this!

**Comment by begandroide** [comment]
@mehmet-erim  thanks to all of you! logic already moved.

**Comment by muhammedaltug** [comment]
This is no longer happening. Tested in 7.0.0-rc.3 version

**Comment by hasanbinboga** [comment]
Hi, I'm Hasan, I'm working as a team leader at Eg Software Company, a part of Eg Group based in Ankara. We are developing applications using Abp Framework. It is really a very useful and comprehensive infrastructure. First of all, we would like to thank you for this. We noticed a situation during development; An instance of the home component is created on the Angular Frontend side. However, when we logged in from the mobile device, we realized that it was loaded twice. I think a similar situation has happened before. As a result of my research, I accessed the following link: https://github.com/abpframework/abp/issues/10781 I think there is a similar situation. We are using version 8.3.1 in our application.
